src/Controllers/SchemaController.js

```diff
@@ -16,6 +16,8 @@
 // TODO: hide all schema logic inside the database adapter.
 // @flow-disable-next
 const Parse = require('parse/node').Parse;
+// @flow-disable-next
+import _ from 'lodash';
```

Comment:
Do we still need lodash? I expect that most of these functions are available natively by now?

Reply:
Mmm, its used on line 1116. Ill check later

Reply:
>Do we still need lodash? I expect that most of these functions are available natively by now?

It still seems to be used a lot around Parse-Server.  We should probably all refactor it at once in a separate PR.